torch_frame/data/tensor_frame.py

```diff
@@ -73,6 +73,13 @@ def __init__(
         self.y = y
         self.validate()
 
+        # Quick mapping from column names into their (stype, idx) pairs in
+        # col_names_dict. Used for fast get_col_feat.
+        self._col_to_stype_idx: Dict[str, Tuple[torch_frame.stype, int]] = {}
+        for stype_name, cols in self.col_names_dict.items():
+            for idx, col in enumerate(cols):
+                self._col_to_stype_idx[col] = (stype_name, idx)
```

Comment:
nit: I'm in favor of a more verbose variable name but feel free to ignore:
```suggestion
        self._col_name_to_stype_idx: Dict[str, Tuple[torch_frame.stype, int]] = {}
        for stype_name, cols in self.col_names_dict.items():
            for idx, col in enumerate(cols):
                self._col_name_to_stype_idx[col] = (stype_name, idx)
```